### PR TITLE
Docs: Fix link to the original Parquet document, add v3.

### PR DESCRIPTION
**Pull request by hedss**
**What this PR does**:
Fixes link to the original Parquet design doc which used Marty's branch.

Also links to the new v3 format.

**Which issue(s) this PR fixes**:
N/A

**Checklist**
- [ ] Tests updated
- [x] Documentation added
- [ ] `CHANGELOG.md` updated - the order of entries should be `[CHANGE]`, `[FEATURE]`, `[ENHANCEMENT]`, `[BUGFIX]`

### CODE REVIEW

File: docs/tempo/website/configuration/parquet.md [hunk 1]
```diff
@@ -53,24 +53,24 @@ storage:
 
 ## Parquet configuration parameters
 
-Some parameters in the Tempo configuration are specific to Parquet.  
+Some parameters in the Tempo configuration are specific to Parquet.
 For more information, refer to the [storage configuration documentation](https://grafana.com/docs/tempo/latest/configuration/#storage).
```
Comment (knylander-grafana) [inline docs/tempo/website/configuration/parquet.md:57]:
```suggestion
For more information, refer to the [storage configuration documentation](/docs/tempo/latest/configuration#storage).
```

File: docs/tempo/website/configuration/parquet.md [hunk 2]
```diff
@@ -10,15 +10,15 @@ weight: 75
 Tempo now has a columnar block format based on Apache Parquet.
 A columnar block format may result in improved search performance and also enables a large ecosystem of tools access to the underlying trace data.
 
-For more information, refer to the [Parquet design document](https://github.com/mdisibio/tempo/blob/design-proposal-parquet/docs/design-proposals/2022-04%20Parquet.md) and [Issue 1480](https://github.com/grafana/tempo/issues/1480).
+For more information, refer to the [Parquet design document](https://github.com/grafana/tempo/blob/main/docs/design-proposals/2022-04%20Parquet.md) and [Issue 1480](https://github.com/grafana/tempo/issues/1480). Additionally, there is now a [Parquet v3 design document](https://github.com/grafana/tempo/blob/main/docs/design-proposals/2023-05%20vParquet3.md), that is included in Tempo 2.2 as an experimental option.
```
Comment (knylander-grafana) [inline docs/tempo/website/configuration/parquet.md:13]:
```suggestion
For more information, refer to the [Parquet design document](https://github.com/grafana/tempo/blob/main/docs/design-proposals/2022-04%20Parquet.md) and [Issue 1480](https://github.com/grafana/tempo/issues/1480). Additionally, there is now a [Parquet v3 design document](https://github.com/grafana/tempo/blob/main/docs/design-proposals/2023-05%20vParquet3.md).
```

**Comment by knylander-grafana** [comment]
I've found some differences between the Parquet.md page you have versus the one that is in my own local copy. I haven't seen what all of the conflicts are reported in the PR yet. If you like, I'm happy to recreate these changes and see if the conflicts persist.

File: docs/tempo/website/configuration/parquet.md (hunk 1 above)
Comment (knylander-grafana) [inline docs/tempo/website/configuration/parquet.md:57]: This is an old link format. It's currently a relref, which makes me think your updates were made to an out-of-date file.

**Comment by knylander-grafana** [comment]
@hedss  I'm going to close this PR. I created your change in this PR: https://github.com/grafana/tempo/pull/2823

**Comment by hedss** [comment]
@knylander-grafana Thank you, I'm sorry that all messed up, typical literally as I'm travelling :(

**Pull request closed without merging**